backend/tracim_backend/app_models/applications.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+import typing
+
+from tracim_backend.app_models.contents import ContentType
+
+
+class Application(object):
+    """
+    Application class with data needed for frontend
+    """
+    def __init__(
+            self,
+            label: str,
+            slug: str,
+            fa_icon: str,
+            is_active: bool,
+            config: typing.Dict[str, str],
+            main_route: str,
+            app_config: 'CFG',
```

Comment:
There is no import of `CFG`. It must be (use if typing.TYPE_CHECKING condition to import it).

Reply:
Done